src/components/DonateMoney.vue

```diff
@@ -25,22 +83,75 @@
 <script lang="js">
 import axios from 'axios'
 export default {
-    name: 'DonateMoney',
-    props: [],
-    data () {
-      return {
-        donation: 1.50
-      }
+  name: 'DonateMoney',
+  props: [],
+  data () {
+    return {
+      donationAmount: 2,
+      customAmountSelected: false,
+      customDonationAmount: undefined, // set min value to be 1.5
```

Comment:
If you're going to use this to set a minimum value, then don't set to undefined. Undefined will cause an exception if customDonationAmount is not set when it is first used.

Reply:
> If you're going to use this to set a minimum value, then don't set to undefined. 

I could remove the comment if that's the source of confusion. It was a deliberate choice to use undefined because I think it makes sense from the user's perspective, that if one was to choose a custom amount, it should be left blank until the user fills it in. The minimum of $1.5 is also a requirement of the feature, as it's the least amount of donation acceptable to offset the cost of sending a letter.

> Undefined will cause an exception if customDonationAmount is not set when it is first used.

That's why `validateDonationAmount` is put in place to handle these exceptions, including when the user is trying to submit an empty value, a value less than $1.50, or a value more than $10,000.00 (I made that maximum limit up. Can be changed to anything that makes more sense.)